packages/qwik-city/buildtime/vite/dev-server.ts

```diff
@@ -35,23 +35,14 @@ import { matchRoute } from 'packages/qwik-city/runtime/src/route-matcher';
 
 export function ssrDevMiddleware(ctx: BuildContext, server: ViteDevServer) {
   const matchRouteRequest = (pathname: string) => {
+    pathname += ctx.opts.trailingSlash && !pathname.endsWith('/') ? '/' : '';
```

Comment:
I am not convinced that this is the same behavior. 
I think the old one checked it twite as it checked the URLs both with and without the trailing slash. The new one does one or the other.

Reply:
You are right, if the behavior is to check with or without then this code is not correct. Anyway [trailingSlash](https://qwik.builder.io/docs/advanced/vite/#trailingslash) is a global vite option, do we need to check both cases?

Reply:
Btw Array.find() is another improvement

Reply:
I would leave the original behavior. The cost of checking the URLs is not that much.

Reply:
With this code I removed one for loop and the behaviour remains the same